### PR TITLE
🚀 feat: ant design gauges for node details & updated condition cards with icons

### LINKED ISSUE #422: Node Gauges
The gauges on the NodeDetails page are outdated and require an update.

Screenshot 2024-07-16 at 12 01 06

To style them in sync with the rest of the application, use the AntDesign library and the [progress components](https://ant.design/components/progress).

### PR DESCRIPTION
**Pull request by ArnavK-09**
closes #422 
related to #440

## 📑 Description
Formatted Nodedetails page
added new ant design gauges
removed old react gauge & its pkg
better condition cards ui with icons
added node creation timestamp

## ✅ Checks
- [x] I have updated the documentation as required
- [x] I have performed a self-review of my code

## ℹ Additional context

image

color pallete preview :-
image

### CODE REVIEW

File: cyclops-ui/src/components/pages/NodeDetails.tsx [hunk 1]
```diff
@@ -399,53 +427,47 @@ const NodeDetails = () => {
...
+              strokeWidth={10}
+              status="normal"
+              percent={resources.cpu * 100}
+              strokeColor={gaugeColors}
             />
             <h1>
-              CPU ({node.requested.cpu}m / {node.available.cpu}m)
+              <strong>CPU</strong>
+              <br />({node.requested.cpu}m / {node.available.cpu}m)
             </h1>
```
Comment (petar-cvit) [inline cyclops-ui/src/components/pages/NodeDetails.tsx:440]: Can we do something like this here? 
```
<h1 style={{marginBottom: "6px"}}>
  <strong>CPU</strong>
</h1>
<h3>
  ({node.requested.cpu}m / {node.available.cpu}m)
</h3>
```
Similarly for other props

File: cyclops-ui/src/components/pages/NodeDetails.tsx [hunk 2]
```diff
@@ -376,6 +396,14 @@ const NodeDetails = () => {
       <Row>
         <Title>{nodeName}</Title>
       </Row>
+      <Row>
+        <Text keyboard>
+          Created On:-{" "}
```
Comment (petar-cvit) [inline cyclops-ui/src/components/pages/NodeDetails.tsx:401]:
```suggestion
          Created on: {" "}
```

**Review by petar-cvit — CHANGES_REQUESTED** [review]
Looks great @ArnavK-09, I left two nit picks

File: cyclops-ui/src/components/pages/NodeDetails.tsx (hunk 2 above)
Comment (ArnavK-09) [reply cyclops-ui/src/components/pages/NodeDetails.tsx]: https://github.com/cyclops-ui/cyclops/pull/443/commits/45d879ac2cacf51a0c1251b0214660c92fd9ff08#diff-3e0644bb482bb090ea078cc63a4529082f10f7215d27fdadd145afc6128afab3R401

File: cyclops-ui/src/components/pages/NodeDetails.tsx [hunk 3]
```diff
@@ -376,6 +396,14 @@ const NodeDetails = () => {
       <Row>
         <Title>{nodeName}</Title>
       </Row>
+      <Row>
+        <Text keyboard>
+          Created on:-{" "}
```
Comment (petar-cvit) [inline cyclops-ui/src/components/pages/NodeDetails.tsx:401]: Can you just remove the dash
```suggestion
          Created on: {" "}
```

**Review by petar-cvit — CHANGES_REQUESTED** [review]

File: cyclops-ui/src/components/pages/NodeDetails.tsx (hunk 3 above)
Comment (ArnavK-09) [reply cyclops-ui/src/components/pages/NodeDetails.tsx]: sorry, forgot to push that, 
https://github.com/cyclops-ui/cyclops/pull/443/commits/1573acc03b28498c089fc3c75eef36b82bbd523f#diff-3e0644bb482bb090ea078cc63a4529082f10f7215d27fdadd145afc6128afab3R401

**Pull request merged**